src/api/app/controllers/staging/staging_projects_controller.rb

```diff
@@ -48,7 +48,7 @@ def copy
 
   def accept
     authorize @staging_project, :accept?
-    authorize @project, :update?
+    authorize @project, :update? if @staging_project.open_unapproved_requests_with_by_project_review
```

Comment:
That should be part of staging project accept policy instead. Skipping authorize calls like this makes them basically untestable as policies - and the whole claim behind not testing controller permissions is that policies in itself are tested (IMO)